pycfmodel/model/resources/properties/property.py

```diff
@@ -5,4 +5,12 @@
 
 
 class Property(CustomModel):
+    """
+    This class is used for all property types that we haven't had time to implement yet.
+
+    Properties:
+
+    - Condition: Define conditions that must comply to apply the condition.
```

Comment:
This may be correct, just double checking this is not a typo :p

Reply:
I just realized my wording is awful in this line. 😂

Reply:
I think that condition is  self-explanatory